src/main/resources/templates/template_hmpps-restricted-patients-api.mustache

```diff
@@ -0,0 +1,24 @@
+<h1>Restricted Patients</h1>
+{{#.}}
+    <br>
+    <table class="data-table">
+        <tr>
+            <td class="data-column-25">Prison number</td>
+            <td class="data-column-25">Discharge time</td>
+            <td class="data-column-25">Hospital location code</td>
+            <td class="data-column-25">Supporting Prison ID</td>
```

Comment:
again here - should this be mapped to description by the restricted patients api?  At present the descriptions are held in Prison API so would require extra calls by the service, but is certainly straightforward to do